src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1350,9 +1350,10 @@ protected override void InvokeDefaultAction(string path)
                     invokeProcess.Start();
                 }
 #elif CORECLR
-                catch (Win32Exception ex) when (ex.NativeErrorCode == 193)
+                catch (Win32Exception ex) when (ex.NativeErrorCode == 193 || ex.NativeErrorCode == 5)
```

Comment:
There's code above this for Unix that is supposed to catch Access Denied for the same situation.  However, `[System.Diagnostics.Process]::Start("/home/steve")` just forks powershell with the new folder.  Need to follow-up with corefx.